components/eventing-controller/pkg/deployment/publisher_deployment.go

```diff
@@ -22,10 +23,11 @@ const (
 	livenessTimeoutSecs      = int32(1)
 	livenessPeriodSecs       = int32(2)
 	bebNamespacePrefix       = "/"
-	instanceLabelKey         = "app.kubernetes.io/instance"
-	instanceLabelValue       = "eventing"
-	dashboardLabelKey        = "kyma-project.io/dashboard"
-	dashboardLabelValue      = "eventing"
+	InstanceLabelKey         = "app.kubernetes.io/instance"
+	InstanceLabelValue       = "eventing"
+	DashboardLabelKey        = "kyma-project.io/dashboard"
+	DashboardLabelValue      = "eventing"
+	BackendTypeLabelKey      = "backendType"
```

Comment:
Kubernetes uses the prefix `app.kubernetes.io` ([source](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/)).
|  Labels without a prefix are private to users.

Moreover, Kyma already comes with the `kyma-project.io` prefix. I suggest to either use this one or go one step further: `eventing.kyma-project.io`. What is your opinion on this ?

Reply:
Or we use `kyma-project.io/eventingBackend`. backendType is very generic and could mean a lot and be used by other pods in Kyma as well in the future 🤔

Reply:
I renamed the label name to `kyma-project.io/eventingBackend` as I think it's more suitable